src/components/Button/Button.tsx

```diff
@@ -1,12 +1,45 @@
 import * as React from 'react';
+import { oneOf } from "prop-types";
```

Comment:
why didn't you get lint rules here? double quotes, spacing inside curly brackets

Reply:
You tell me. fixed.